src/main/java/com/google/javascript/gents/TypeAnnotationPass.java

```diff
@@ -108,6 +108,25 @@ public void visit(NodeTraversal t, Node n, Node parent) {
             setTypeExpression(n, bestJSDocInfo.getReturnType(), true);
           }
           break;
+        case CLASS:
+          if (bestJSDocInfo != null) {
+            List<JSTypeExpression> interfaces = bestJSDocInfo.getImplementedInterfaces();
+            if (!interfaces.isEmpty()) {
+              Node impls = new Node(Token.IMPLEMENTS);
+              for (JSTypeExpression type : interfaces) {
+                impls.addChildToBack(convertTypeNodeAST(type.getRoot()));
```

Comment:
@mprobst Oh,It's my fault,my nickname is a keyword,I misunderstood it.I'll change my nickname.⊙﹏⊙sorry～